libs/util/http.client.js

```diff
@@ -18,7 +18,7 @@ var _ = require('lodash'),
         timeout: 3000,
         retry: {
             interval: 200,
-            max_retries: 2
+            max_retries: 0
```

Comment:
Turning retry off to keep backward compatible behavior of no automatic retry, which was due to a bug that is fixed in this commit.